### ISSUE TITLE
How to list all colormaps supported by the package?

### ISSUE THREAD

**Issue by wxguy** [pratiman-91/colormaps#9]
Is there a way to list all the supported colormaps by the package as being done for matplotlib with `plt.colormaps()` command?

**Comment by wxguy** [comment]
Right now I check `colormaps` attribute with `dir(cmaps)` and check if it is not starting with `_` and has `colors` name. It works and gives me what I want. But I am sure that it is easy to implement in your package as one of the features.

**Comment by pratiman-91** [comment]
@wxguy it is a good idea. i can add a similar function. Thanks!

**Comment by pratiman-91** [comment]
added two new functions: (will be available from `version 0.4.1`) 

1. Show different collections: for example, to show all possible collection of `cmasher`
```
from colormaps.utils import show_cmaps_collection
show_cmaps_collection(collection='cmasher')
```
2. Show all possible collections:
```
from colormaps.utils import show_cmaps_all
show_cmaps_all()
```
